compiler/circle-execution-plan/src/ExecutionPlanner.h

```diff
@@ -19,6 +19,7 @@
 
 #include <luci/IR/Module.h>
 #include <luci/Plan/CircleNodeExecutionPlan.h>
+#include "TargetPlatform.h"
```

Comment:
let's add headers this way:
```suggestion
#include "TargetPlatform.h"
#include <luci/IR/Module.h>
#include <luci/Plan/CircleNodeExecutionPlan.h>
```

Headers more related to current component should be placed higher, like:
``` c++
#include "header from this component"
#include <header from related component>
#include <header from third party project>
#include <system header>
```

This approach helps to detect issues with incomplete headers. For example cases where local header uses std library, but do not include it properly.

Reply:
Done